src/main/java/thestonedturtle/lootlogger/LootLoggerConfig.java

```diff
@@ -41,6 +42,17 @@ default ItemSortTypes itemSortType()
 
 	@ConfigItem(
 		position = 2,
+		keyName = "itemMissingAlpha",
+		name = "Missing item opacity",
+		description = "Opacity for image of an unacquired item (0 being transparent and 100 being opaque, default 35)"
```

Comment:
Change text to: "Controls the image opacity for unacquired unique items inside the Uniques panel.<br/> 0 is transparent and 100 is fully opaque, default 35"